templates/address-space-controller/020-ConfigMap-standard-broker-definitions.yaml

```diff
@@ -483,3 +595,11 @@ data:
         a quarter of BROKER_MEMORY_LIMIT
       name: GLOBAL_MAX_SIZE
       value: "-1"
+    - description: The name of the broker
+      displayName: AMQ Name
+      name: AMQ_NAME
+      value: broker
+    - description: Home directory to for the broker instance
+      displayName: Home
+      name: HOME
+      value: /home/artemis
```

Comment:
Remove template parameters and specify values in pod container env instead.

Reply:
now removed